benchmark/src/main/kotlin/org/virtuslab/akkasaferserializer/benchmark/Plot.kt

```diff
@@ -0,0 +1,120 @@
+package org.virtuslab.akkasaferserializer.benchmark
+
+import kotlinx.html.FlowContent
+import space.kscience.plotly.*
+import space.kscience.plotly.models.Bar
+import space.kscience.plotly.models.BarMode
+import space.kscience.plotly.palettes.Xkcd
+import java.io.File
+
+fun main() {
```

Comment:
Actually, since that's the only Kotlin source out there... can it be easily converted to Scala? 🤔

Reply:
(I'm afraid the closures might be hard to get right tho)